README.md

```diff
@@ -1,30 +1,30 @@
 # Mysite
 ## A project in Django (Python 3.x)
-**A simple blogging platform for the modern era.**
+**_A simple blogging platform for the modern era._**
 
-This web application generates a simple but capable blog using Django. The platform allows authors to create text posts using a dedicated admin portal, and allows logged in users to post comments through a simple comment form. 
+####This web application generates a simple but capable blog using Django. The platform allows authors to create text posts using a dedicated admin portal, and allows logged in users to post comments through a simple comment form. 
```

Comment:
Github requires a space after the heading hashtags to render properly as can be seen in the preview. Please include this to render properly.